ui/cap-react/src/components/search/SearchFacet/SearchFacet.js

```diff
@@ -54,96 +58,113 @@ const SearchFacet = ({
-                      direction="row"
-                      align="center"
-                      style={{
-                        fontSize: "0.8em"
-                      }}
-                    >
-                      <CheckBox
-                        label={`${field.key} ${
-                          typeof field.doc_count === "object"
-                            ? `(${field.doc_count.doc_count})`
-                            : `(${field.doc_count})`
-                        }`}
-                        key={field.key}
-                        name={String(field.key)}
-                        checked={
-                          isAggSelected(selectedAggs[category], field.key)
-                            ? true
-                            : false
-                        }
-                        onChange={onChange(category)}
-                      />
-                    </Box>
-                    <Box
-                      style={{
-                        margin: "5px 0"
-                      }}
-                      margin={{
-                        left: "small"
-                      }}
-                    >
-                      {isAggSelected(selectedAggs[category], field.key) &&
-                        Object.keys(field)
-                          .filter(key => key.startsWith("facet_"))
-                          .map(key => {
-                            return field[key].buckets.map(nested_field => (
-                              <Box
-                                size="medium"
-                                key={String(nested_field.key)}
-                                direction="row"
-                                align="start"
-                                style={{
-                                  fontSize: "0.8em"
-                                }}
-                              >
-                                <CheckBox
-                                  id="search_checkbox"
-                                  label={nested_field.key}
-                                  key={nested_field.key}
-                                  name={String(nested_field.key)}
-                                  checked={
-                                    isAggSelected(
-                                      selectedAggs[key.replace("facet_", "")],
-                                      nested_field.key
-                                    )
-                                      ? true
-                                      : false
-                                  }
-                                  onChange={onChange(key.replace("facet_", ""))}
-                                />
-                                <Box align="end">
-                                  {typeof nested_field.doc_count === "object"
-                                    ? nested_field.doc_count.doc_count
-                                    : nested_field.doc_count}
+          category === "next_deadline_date" ? (
```

Comment:
shouldn't be explicit to a key (e.g. "deadlineDateValues"). instead we should check the `meta.type` info to distinguish on facet display type